src/rpc/handlers/ServerInfo.h

```diff
@@ -223,6 +244,10 @@ class BaseServerInfoHandler
             jv.as_object()["etl"] = info.adminSection->etl;
             jv.as_object()[JS(counters)] = info.adminSection->counters;
             jv.as_object()[JS(counters)].as_object()["subscriptions"] = info.adminSection->subscriptions;
+            if (!info.adminSection->backend_counters.empty())
```

Comment:
Don't we want to return zeroes at the start? chances are low but what if a script will expect the values there but at the very start there would be none?

Reply:
What do you mean? Here we are serialising `InfoSection` into json and checking whether `InfoSection` has backend counters because it will have it only if there was `backend_counters: true` parameter in request. I could make backendCounters `std::optional<boost::json::object>` to better express the logic here.